build_local_windows.ps1

```diff
@@ -53,6 +51,48 @@ python -m venv "$tmpdir/dcos_build_venv"
 
 # Install the DC/OS tools
 ./prep_local_windows.ps1
+# Build a release of DC/OS
+release create $env:USERNAME local_build windows
+
+rm -fo dcos-release.config.yaml
+
+# Creating temp dir, for instance:
+$local_artifacts_dir = "./packages/cache"
+mkdir -f $local_artifacts_dir
+
+##Write DCOS installer locally
+$config_yaml =
+"storage: `
+   local: `
+    kind: local_path `
+    path: $local_artifacts_dir `
+options: `
+  preferred: local `
+  cloudformation_s3_url: https://s3-us-west-2.amazonaws.com/downloads.dcos.io/dcos"
+
+   $config_yaml | Set-Content -Path "dcos-release.config.yaml"
+
+type dcos-release.config.yaml
+# Create a python virtual environment to install the DC/OS tools to
+python -m venv "$tmpdir/dcos_build_venv"
+. "$tmpdir/dcos_build_venv/Scripts/Activate.ps1"
 
+# Install the DC/OS tools
+./prep_local_windows.ps1
 # Build a release of DC/OS
 release create $env:USERNAME local_build windows
+
+# Build tar ball for windows. 2 params: packages location and DC/OS variant:
+powershell.exe -NoProfile -ExecutionPolicy Bypass -Command "& .\build_genconf_windows.ps1 '$local_artifacts_dir\testing'"
+
+# Import AWS modules on Azure TeamCity runner
+Install-Module -Name AWS.Tools.Common -Force
+Install-Module -Name AWS.Tools.S3 -Force
+
+# Set and Read AWS Credentials:
+Set-AWSCredential -AccessKey $env:AWS_ACCESS_KEY_ID -SecretKey $env:AWS_SECRET_ACCESS_KEY -StoreAs aws_s3_windows
+Set-AWSCredential -ProfileName aws_s3_windows
+# Upload Tar Ball to dcos.download.io
+Write-S3Object -BucketName "downloads.dcos.io" -Key "dcos\testing\$env:BRANCH\dcos_generate_config_win.sh" -File ".\dcos_generate_config_win.sh" -CannedACLName public-read
```

Comment:
This should be

```Write-S3Object -BucketName "downloads.dcos.io" -Key "dcos\testing\'$env:TEAMCITY_BRANCH'\dcos_generate_config_win.sh" -File ".\dcos_generate_config_win.sh" -CannedACLName public-read```

And we should be good to see the upload happening.

Reply:
Example of the change is here: https://github.com/dcos/dcos/pull/6476 - where the upload is also verified.